File: php/inputoutput.php
<?php

echo "Masukkan nama barang: ";
$nama = trim(fgets(STDIN));

echo "Masukkan harga barang: ";
$harga = floatval(trim(fgets(STDIN)));

echo "Masukkan jumlah beli: ";
$jumlah = intval(trim(fgets(STDIN)));

$total = $harga * $jumlah;

if ($total > 16100) {
    $diskon = $total * 0.01;
} else {
    $diskon = 0;
}

$total_bayar = $total - $diskon;

echo "\n===== STRUK PEMBELIAN =====\n";
echo "Nama Barang     : $nama\n";
echo "Harga Satuan    : $harga\n";
echo "Jumlah Beli     : $jumlah\n";
echo "Total Harga     : $total\n";
echo "Diskon (1%)     : $diskon\n";
echo "Total Bayar     : $total_bayar\n";
echo "===========================\n";

File: c atau c++/inputoutput.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string nama;
    double harga;
    int jumlah;

    cout << "Masukkan nama barang: ";
    getline(cin, nama);

    cout << "Masukkan harga barang: ";
    cin >> harga;

    cout << "Masukkan jumlah beli: ";
    cin >> jumlah;

    double total = harga * jumlah;
    double diskon;

    if (total > 16100) {
        diskon = total * 0.01;
    } else {
        diskon = 0;
    }

    double total_bayar = total - diskon;

    cout << "\n===== STRUK PEMBELIAN =====\n";
    cout << "Nama Barang     : " << nama << endl;
    cout << "Harga Satuan    : " << harga << endl;
    cout << "Jumlah Beli     : " << jumlah << endl;
    cout << "Total Harga     : " << total << endl;
    cout << "Diskon (1%)     : " << diskon << endl;
    cout << "Total Bayar     : " << total_bayar << endl;
    cout << "===========================\n";

    return 0;
}